sycl/include/CL/sycl/info/info_desc.hpp

```diff
@@ -154,6 +154,7 @@ enum class device : cl_device_info {
   atomic64 = PI_DEVICE_INFO_ATOMIC_64,
   atomic_memory_order_capabilities =
       PI_DEVICE_INFO_ATOMIC_MEMORY_ORDER_CAPABILITIES,
+  ext_intel_bf16_conversion,
```

Comment:
Are we going to map it to a specific value in future?

Reply:
Yes